syncer/db.go

```diff
@@ -238,7 +238,7 @@ func (conn *DBConn) executeSQLWithIgnore(tctx *tcontext.Context, ignoreError fun
 				sqlRetriesTotal.WithLabelValues("stmt_exec", conn.cfg.Name).Add(1)
 				return true
 			}
-			if retry.IsRetryableErrorFastFailFilter(err) {
+			if retry.IsRetryableError(err) {
```

Comment:
why change to this function? seems change retry logic for ddl

Reply:
`IsRetryableErrorFastFailFilter` as a black-list for `1105`, but we have changed to use a white-list for `1105` now.

so, the _fast fail error_ in `IsRetryableErrorFastFailFilter` will not be retry in `IsRetryableError`.